src/actions/index.js

```diff
@@ -20,3 +19,5 @@ export const exerciseZeroFormSubmit = (submit) => {
         },
     };
 };
+
+// Meet me in ../reducers/exercise0.js once you've understood this file
```

Comment:
Moved the "meet me in" comments to the bottom of the file, that way people have to read through the whole file before they get to it